### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by prathameshpcodes**

### CODE REVIEW

File: css/pages/cart.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  max-width: 45rem;
+  overflow: hidden;
+}
+
+.cart-btn-qty {
+  border: thin solid var(--color-slate-300);
+  border-radius: 10rem;
+  color: var(--color-slate-800);
+  font-size: 1.2rem;
+  margin: 1rem 0.5rem;
```
Comment (varun1233) [inline css/pages/cart.css:11]: Use px for values less than 1 rem

**Comment by vivvvek** [comment]
increase the font size of Add to cart page, otherwise you are good to go

**Comment by prathameshpcodes** [comment]
> increase the font size of Add to cart page, otherwise you are good to go

ok will take care of that

File: pages/cart.html [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+              </div>
+            </div>
+          </li>
+
+          <li class="mb-1 shadow-light">
+            <div class="card cart-card">
+              <img
+                loading="lazy"
+                class="max-w-20"
+                src="/assets/demo-product-1.jpg"
```
Comment (varun1233) [inline pages/cart.html:198]: Provide alt for images, it's considered as a good practice

Comment (prathameshpcodes) [reply pages/cart.html:198]: sure,thanks for the suggestion

**Review by vivvvek — APPROVED** [review]

**Comment by kaustubh-jsr** [comment]
Apart from the above suggested changes, I don't see a problem. Looks good to merge.

**Review by kaustubh-jsr — APPROVED** [review]

**Pull request merged**